compiler-core/src/type_/tests.rs

```diff
@@ -1663,6 +1663,24 @@ fn unused_variable_warnings_test() {
     assert_no_warnings!("pub fn a(b) { case b { #(c, _) -> c } }");
 }
 
+#[test]
```

Comment:
There was a test on the JS suite that showed this was an accepted syntax.
I re-created the test in compiler core, and the need for ImportedNameSrcSpan is because of this.

Reply:
Do you think we should force them to merge the two imports into one?

Reply:
No. IMHO this is a linter's job.

For instance, I use a linter to prevent me from doing it in an important .py file, but I regularly import stuff like this example in jupyter notebook or in interactive mode.

Reply:
I think different rules would likely apply in a future interactive mode so we can put that aside for now. These rules will be only for regular source files.

Reply:
`make test` fails because it can't compile stdlib https://github.com/gleam-lang/stdlib/blob/main/src/gleam/dynamic.gleam#L3-L4

Reply:
Should this syntax be valid? :thinking:

Reply:
Good find. I've released a new version of the stdlib with that mistake corrected. If you rebase on main you will get the fix

Reply:
I did. Now it happens on https://github.com/gleam-lang/stdlib/blob/main/src/gleam/string.gleam#L8-L9